bitnami/jaeger/templates/_helpers.tpl

```diff
@@ -0,0 +1,121 @@
+{{/* vim: set filetype=mustache: */}}
+
+
+{{/*
+Return the proper jaeger image name
+*/}}
+{{- define "jaeger.image" -}}
+{{ include "common.images.image" (dict "imageRoot" .Values.image "global" .Values.global) }}
+{{- end -}}
+
+{{/*
+Return the proper cassandra external image name
+*/}}
+{{- define "jaeger.cqlshImage" -}}
+{{ include "common.images.image" (dict "imageRoot" .Values.cqlshImage "global" .Values.global) }}
+{{- end -}}
+
+
+{{/*
+Create the name of the query deployment
+*/}}
+{{- define "jaeger.query.fullname" -}}
+    {{ printf "%s-query" (include "common.names.fullname" .) }}
+{{- end -}}
+
+{{/*
+Create a container for checking cassandra availability
+*/}}
+{{- define "jaeger.waitForDBInitContainer" -}}
+- name: jaeger-cassandra-ready-check
+  image: {{ include "jaeger.cqlshImage" . }}
+  command:
+    - /bin/bash
+  args:
+    - -ec
+    - |
+      #!/bin/bash
+
+      set -o errexit
+      set -o nounset
+      set -o pipefail
+
+      . /opt/bitnami/scripts/libos.sh
+
+      check_cassandra_keyspace_schema() {
+          echo "SELECT 1" | cqlsh -u $CASSANDRA_USERNAME -p $CASSANDRA_PASSWORD -e "SELECT COUNT(*) FROM ${CASSANDRA_KEYSPACE}.traces"
+      }
+
+      info "Connecting to the Cassandra instance $CQLSH_HOST:$CQLSH_PORT"
+      if ! retry_while "check_cassandra_keyspace_schema" 12 30; then
+        error "Could not connect to the database server"
+        exit 1
+      else
+        info "Connection check success"
+      fi
+  env:
+  - name: CQLSH
+    value: /opt/bitnami/cassandra/bin/cqlsh
+  - name: CQLSH_HOST
+    value: {{ printf "%s-cassandra" (include "common.names.fullname" .) }}
+  - name: CQLSH_PORT
+    value: {{ .Values.cassandra.containerPorts.cql | quote }}
```

Comment:
I would expect here something like:

1) If we are using the subchart, use the service port, not the container port
2) If we are using the external cassandra, use the port provided by the user

This is normally done inside a helper, check other charts like Appsmith

Reply:
Missed that one [259761e](https://github.com/bitnami/charts/pull/13480/commits/259761ebaded19e9049b739f4cdaaf6986493827)